src/Service/SshDiagnostics.php

```diff
@@ -205,7 +205,7 @@ public function diagnoseFailure($uri, $exitCode, Process $failedProcess = null,
                 $this->stdErr->writeln('');
             }
             $this->stdErr->writeln(sprintf(
-                'You probably need to add an SSH key, with: <comment>%s ssh-key:add</comment>',
+                '<error>[Could not connect]</error> Either the app is being deployed or you need to add an SSH key, with: <comment>%s ssh-key:add</comment>',
```

Comment:
I'm wondering if there might be more possibilities at this stage. Maybe if we can't provide accurate advice here we should remove the SSH key hint entirely.